src/lib/components/Pagination/Pagination.js

```diff
@@ -19,7 +19,7 @@ class Pagination extends Component {
       ...extraParams
     } = this.props;
     // Maximum number of pages allowed (backend window limitation)
-    const maximumWindowPages = Math.ceil(
+    const maximumWindowPages = Math.floor(
```

Comment:
isn't it that this lead to a search error when clicking on the last page now?

Reply:
no this fixes it, it was my fault